src/Kaba.ts

```diff
@@ -274,12 +274,24 @@ export class Kaba
 
         if (Object.keys(this.jsEntries).length)
         {
+            const compilerConfigs: kaba.WebpackBuildConfig[] = [];
+
+            Object.keys(this.jsEntries).forEach((entry: string) => {
+                if (this.buildModern)
+                {
+                    compilerConfigs.push(this.buildWebpackConfig(entry, this.jsEntries[entry], cliConfig, true));
+                }
+            });
+
+            // The log output gets really big with multiple entry files.
+            // Therefore the linted (legacy) entries are added after the modern ones.
```

Comment:
We should combine this one and use less logging for everything except the legacy builds

Reply:
And then combine them in a single list